### PR TITLE
Fix symbol replacement after loop merge

### PR DESCRIPTION
**Pull request by FabioLuporini**

### CODE REVIEW

**Comment by kynan** [comment]
Travis seems to be unhappy.

File: coffee/base.py [hunk 1]
```diff
@@ -445,6 +445,9 @@ def __init__(self, symbol, rank=(), offset=()):
     def operands(self):
         return [self.symbol, self.rank, self.offset], {}
 
+    def reconstruct(self):
+        return Symbol(self.symbol, self.rank, self.offset)
```
Comment (wence-) [inline coffee/base.py]: `reconstruct` should take new arguments to reconstruct with.  The default reconstruct method does this.  Why do you need this change (which causes tests to fail)?

Comment (FabioLuporini) [inline coffee/base.py]: Ah right, I think I see now how this works. I'll fix this

**Comment by FabioLuporini** [comment]
uops! That was due to leftover code, I have no idea of why it was there. Anyway, purged

**Comment by wence-** [comment]
This is good to go, right?

**Pull request merged**